srcpkgs/chatterino2/template

```diff
@@ -1,55 +1,56 @@
 # Template file for 'chatterino2'
 pkgname=chatterino2
-version=2.2.2
-revision=2
-_signals_commit=6665ccad90461c01b7fe704a98a835953d644156
-_settings_commit=a5040463c01e6b0e562eab82e0decb29cab9b450
-_humanize_commit=4e00a03623966723f23ca3034c1ad944009cd7be
-_serialize_commit=130ffc3ec722284ca454a1e70c5478a75f380144
-build_style=qmake
-configure_args="INCLUDEPATH+=$XBPS_CROSS_BASE/usr/include/qt5/Communi/IrcCore
- INCLUDEPATH+=$XBPS_CROSS_BASE/usr/include/qt5/Communi/IrcModel
- INCLUDEPATH+=$XBPS_CROSS_BASE/usr/include/qt5/Communi/IrcUtil"
-hostmakedepends="qt5-qmake qt5-host-tools pkg-config"
-makedepends="qt5-svg-devel qt5-multimedia-devel
- boost-devel rapidjson libcommuni-devel websocketpp
- qtkeychain-qt5-devel"
-short_desc="Qt-based twitch chat client"
-maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
-license="MIT"
+version=2.3.3
+revision=1
+_WinToast_commit="ce441336ef057576dfb377be754bc5357a04ff85"
+_libcommuni_commit="ef8daa14946b8e19f536200e28db2b25e8311ba5"
+_qBreakpad_commit="a4626c12e9ae6f02fc1ca7a4e399bd8307424103"
+_serialize_commit="7d37cbfd5ac3bfbe046118e1cec3d32ba4696469"
+_settings_commit="7cf8431d644332107a51a46c1e3de70e64692f0c"
+_signals_commit="baf5bb04bd13b090e405e0447c89a811f7e23ddc"
+_qtkeychain_commit="de954627363b0b4bff9a2616f1a409b7e14d5df9"
+_sanitizers_cmake_commit="99e159ec9bc8dd362b08d18436bd40ff0648417b"
+build_style=cmake
+hostmakedepends="qt5-host-tools"
```

Comment:
Try adding `qt5-qmake` for the cross builds.

Reply:
Chatterino doesn't fully work on arm at the moment and only builds with qt6

Reply:
Should chatterino have any code that's arch specific?

Do you mean that we need to build with qt6 for ARM to work? That is an option, since qt6 is in repos.

Reply:
Anyway, please add `qt5-qmake` so it can get further in the build for errors to appear.